--- origsrc/qtwebkit-opensource-src-5.2.1/Tools/qmake/mkspecs/features/default_pre.prf	2014-02-01 14:37:41.000000000 -0600
+++ src/qtwebkit-opensource-src-5.2.1/Tools/qmake/mkspecs/features/default_pre.prf	2014-04-10 18:15:09.927933100 -0500
@@ -106,7 +106,7 @@ if(win32|mac):!macx-xcode {
 # A newer version of flex is required on Windows. At the moment the only
 # one that appears to provide binaries and is not cygwin is winflex.
 FLEX = flex
-win32: FLEX = win_flex
+contains(QMAKE_HOST, win32): FLEX = win_flex
 
 BIN_EXTENSION =
 win32: BIN_EXTENSION = .exe
